src/uct/api/uct.h

```diff
@@ -340,8 +340,21 @@ enum uct_md_mem_flags {
                                                 mapping may be deferred until
                                                 it is accessed by the CPU or a
                                                 transport. */
-    UCT_MD_MEM_FLAG_FIXED    = UCS_BIT(1)  /**< Place the mapping at exactly
+    UCT_MD_MEM_FLAG_FIXED    = UCS_BIT(1), /**< Place the mapping at exactly
                                                 defined address */
+    /* memory access flags */
+    UCT_MD_MEM_ACCESS_REMOTE_PUT    = UCS_BIT(2), /**< enable remote put access */
+    UCT_MD_MEM_ACCESS_REMOTE_GET    = UCS_BIT(3), /**< enable remote get access */
+    UCT_MD_MEM_ACCESS_REMOTE_ATOMIC = UCS_BIT(4), /**< enable remote atomic access */
+
+    /**< enable local and remote access for all operations */
+    UCT_MD_MEM_ACCESS_ALL =  (UCT_MD_MEM_ACCESS_REMOTE_PUT|
+                              UCT_MD_MEM_ACCESS_REMOTE_GET|
+                              UCT_MD_MEM_ACCESS_REMOTE_ATOMIC),
+
+    /**< enable local and remote access for the put and get */
```

Comment:
1. Pls remove the angle bracket '<' so that the comment aligns properly with the name
2. s/for the put and get/for put and get operations/